geopandas/array.py

```diff
@@ -132,7 +133,7 @@ def _is_scalar_geometry(geom):
         return isinstance(geom, BaseGeometry)
 
 
-def from_shapely(data, crs=None):
+def from_shapely(data: list, crs=None) -> "GeometryArray":
```

Comment:
Why not directly uses `GeometryArray`?
There are all in the same script `array.py`

Reply:
GeometryArray is not completely defined by line 136, so I used string literals: https://www.python.org/dev/peps/pep-0484/#forward-references

Reply:
When I directly used `GeometryArray`, my tests failed because it wasn't defined by then.

Reply:
And to solve `NameError: name 'GeometryArray' is not defined`, there should move `class GeometryArray` to top.

Reply:
@Zeroto521 I don't see a `NameError`

Reply:
> When I directly used GeometryArray, my tests failed because it wasn't defined by then.

To solve this problem, there should move `class GeometryArray` to top.